components/social-media-posts/RedditPostComponent.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import SocialMediaButtonsContainer from 'components/SocialMediaButtonsContainer'
+import Headline from "components/Headline"
+import AvatarComponent from "components/AvatarComponent"
+import { jsx } from '@emotion/react'
+
+interface Props {
+  avatarAlt: string;
+  avatarChildren: string;
+  avatarSrc: string;
+  children: React.ReactNode;
+  className: string;
```

Comment:
let's remove all of the props except for children in favor of hardcoding values into the avatar component